README.md

```diff
@@ -61,7 +62,7 @@ All content must start with a Content Proposal. This will be in the form of a Gi
 
 ### GitHub Workflow
 
-Once your proposal has been moved to the `accepted` stage, you can begin working on your content. Here are the steps to do this:
+Once your proposal has been moved to the `approved` stage, you can begin working on your content. Here are the steps to do this:
```

Comment:
Just a minor detail (that is probably outside of the scope of this PR): Currently, it seems like the labels on the issues aren't followed that closely.
Specifically, I haven't seen [any label after `approved`](https://github.com/java/devjava-content/issues?q=is%3Aissue+is%3Aclosed) being used.